index.html

```diff
@@ -44,5 +44,178 @@ <h2><img src="image/LET’S CONNECT.svg" alt=""></h2>
+                    sign-ups required.
+                </p>
+                <ul class="tech">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+
+            <div class="my-work">
+                <img src="image/Portfolio8.svg" alt="image4" />
+                <h2>Multi-Post Stories</h2>
+                <div class="position-container">
+                    <span>Canopy</span>
+                    <ul>
+                        <li class="developed marked">Back End Dev</li>
+                        <li class="year marked">2022</li>
+                    </ul>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required.
+                </p>
+                <ul class="tech skills">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+        </section>
+        <section class="section3">
+            <div class="about">
+                <h1>About Myself</h1>
+                <p>
+                    Hello I’m a software developer! I can help you build a product,
+                    feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.
+                </p>
+
+                <div class="social">
+                    <p>LET’S CONNECT</p>
+                </div>
+
+                <div class="icons">
+                    <ul>
+                        <li>
+                            <a href="#"><i class="fa fa-facebook fa-2x"></i></a>
+                        </li>
+                        <li>
+                            <a href="#"><i class="fa fa-twitter fa-2x"></i></a>
+                        </li>
+                        <li>
+                            <a href="#"><i class="fa fa-instagram fa-2x"></i></a>
+                        </li>
+
+                        <li>
+                            <a href="#"><i class="fa fa-github fa-2x"></i></a>
+                        </li>
+
+                        <li>
+                            <a href="#"><i class="fa fa-linkedin fa-2x"></i></a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <div class="get-resume">
+                <button type="button" class="get-resume-btn">Get My Resume</button>
+            </div>
+
+            <div class="accordion">
+                <ul class="mother-accordion">
+                    <li class="accordion-item">
+                        <h2>
+                            <span>Languages</span>
```

Comment:
It looks amazing 👌but you can improve it by following the exact Figma design. Please make sure to fix the font-weight here. These kinds of simple things can improve your design significantly.

![image](https://user-images.githubusercontent.com/34779928/198817838-ac39db1f-2743-446f-9e2a-e2b12ef3f03f.png)